_config.yml

```diff
@@ -28,7 +28,11 @@ exclude: ["Gemfile",
           "scripts/",
           ".gitignore",
           ".sass-cache/",
-          ".idea/"]
+          ".idea/",
+          "README.md",
+          ".ruby-version",
+          "_includes/site",
```

Comment:
This has the html files.  Can you explain what they are used for, and why they should be excluded?

Reply:
When the website is built, static site is generated and sent to a directory called `_site`. If `_site` is deleted it can be regenerated by building website as this is generated every time website is built. Hence, `_site` is not used and it need not be stored on GitHub. Tested changes on local and it works just fine.

Reply:
The excluded resource is `_includes/site`, instead of `_site`.  Are we discussing the same thing?

Also, there are some references like this... is this referring to  `_includes/site`?  And if so, does the page that use this include still work?  See the screenshot below.
'{% include site/news/news_item.html post=post %}`

![image](https://user-images.githubusercontent.com/6977544/70581603-8b8e5400-1b6c-11ea-80fb-44168e8f36b1.png)

Reply:
By default, all the files in your in your Jekyll project are included in the output. In `_config.yml` file we can exclude files and directories that shouldn't be included in the output which Jekyll will then ignore. [Here](https://jekyllrb.com/docs/configuration/options/) is the documentation on `exclude` straight from the Jekyll site.